src/dataflow/src/sink/kafka.rs

```diff
@@ -26,10 +27,14 @@ use rdkafka::error::{KafkaError, RDKafkaErrorCode};
 use rdkafka::message::Message;
 use rdkafka::producer::Producer;
 use rdkafka::producer::{BaseRecord, DeliveryResult, ProducerContext, ThreadedProducer};
+use timely::dataflow::channels::pact::Exchange as ExchangePact;
```

Comment:
I don't think you need to rename this import

```
diff --git a/src/dataflow/src/sink/kafka.rs b/src/dataflow/src/sink/kafka.rs
index 1629feb0a..6681f8610 100644
--- a/src/dataflow/src/sink/kafka.rs
+++ b/src/dataflow/src/sink/kafka.rs
@@ -27,8 +27,7 @@ use rdkafka::error::{KafkaError, RDKafkaErrorCode};
 use rdkafka::message::Message;
 use rdkafka::producer::Producer;
 use rdkafka::producer::{BaseRecord, DeliveryResult, ProducerContext, ThreadedProducer};
-use timely::dataflow::channels::pact::Exchange as ExchangePact;
-use timely::dataflow::channels::pact::Pipeline;
+use timely::dataflow::channels::pact::{Exchange, Pipeline};
 use timely::dataflow::operators::generic::builder_rc::OperatorBuilder;
 use timely::dataflow::operators::generic::{FrontieredInputHandle, OutputHandle};
 use timely::dataflow::operators::Capability;
@@ -692,7 +691,7 @@ where
     let name = format!("{}-sort_on_one_worker", sink_name);

     let sorted_output_stream = input_stream.unary_frontier(
-        ExchangePact::new(move |_| sink_id.hashed()),
+        Exchange::new(move |_| sink_id.hashed()),
         &name,
         move |_, _| {
             let mut stash = HashMap::new();
```
compiles for me

Reply:
Ah yes, this was from another version where I also had also imported the `Exchange` operator (https://github.com/TimelyDataflow/timely-dataflow/blob/221f98930c92b75d215c720e88f559cdab7d9147/timely/src/dataflow/operators/exchange.rs#L9)